src/kagglehub/cache.py

```diff
@@ -4,9 +4,10 @@
 from typing import Optional
 
 from kagglehub.config import get_cache_folder
-from kagglehub.handle import CompetitionHandle, DatasetHandle, ModelHandle, ResourceHandle
+from kagglehub.handle import CodeHandle, CompetitionHandle, DatasetHandle, ModelHandle, ResourceHandle
```

Comment:
I think we should stick with the term notebook here to be consistent. 

and ditto in other areas.

Reply:
We could also use Kernel (which is what we use usually in our codebase).

Reply:
kernel is used internally, notebook is the "public term".
i think since this is public repo, we should stick to notebook. 

using it interchangeably in the same repo, will get pretty confusing for both us and external users.

Reply:
following that distinction, should the public api endpoint also fall under `/api/v1/notebooks/`?

Reply:
public api endpoint should remain as kernel, i meant code written in kagglehub like "CodeHandle" and "CODE_CACHE_SUBFOLDER"

Reply:
updated everything to say "notebooks" 1876af0eebe9f987dc265f17339289ee8cd9d31b